src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -532,7 +528,7 @@ private void testResponseRate() {
     
     private InstructorFeedbacksPage navigateToInstructorFeedbacksPage() {
         
-        AppUrl feedbacksPageUrl = createUrl(Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE).withInstructorId(instructorId);
+        AppUrl feedbacksPageUrl = createUrl(Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE).withUserId(instructorId);
```

Comment:
@kanghj sorry for the delay! I changed it as loading the url with the instructor id rather than the user id generates the wrong session table (on the staging/live server) resulting in other test failures. I fixed it under this issue as I wanted to take care of the other points of failure for this test on the staging/live server in this PR as well. I will update the issue to include the other areas with the other test failure locations.